### ISSUE TITLE
Cleanup profraw files + `--no-clean`

### ISSUE THREAD

**Issue by SamChou19815** [taiki-e/cargo-llvm-cov#292]
I really like the `--no-clean` flag since it makes incremental re-run of tests much faster. However, it eventually gets slower and slower since there are tons of profraw files generated.

In practice, I found that running `rm -f target/llvm-cov-target/*.profraw` before a `llvm-cov` run with `--no-clean` flag has the best of both world: the `target/llvm-cov-target` won't blow up in size, and we still get all the incremental runs' benefit.

Could this be added as a builtin flag so I don't have to write a custom script to do this?

**Comment by taiki-e** [comment]
Sounds reasonable to me.

**Comment by TroyKomodo** [comment]
Do incremental builds actually work? 
```bash
$ cargo llvm-cov show-env
...
CARGO_INCREMENTAL="0"
...
```

**Comment by taiki-e** [comment]
@TroyKomodo 
--no-clean is unrelated to the incremental compilation of rustc itself, but improves the performance of successive builds, since the cleanup to prevent false positives in several situations that occur before the build and the rebuilds triggered by the cleanup are no longer performed.

CARGO_INCREMENTAL=0 was needed at some point in the past, but may no longer be needed with the [cleanup in 0.1.5](https://github.com/taiki-e/cargo-llvm-cov/commit/395b95afd7a22aa62a1b27ab5843282d6f1a8be4).

**Comment by smoelius** [comment]
> Could this be added as a builtin flag so I don't have to write a custom script to do this?

I'm considering implementing this. What should an invocation look like?

**Comment by taiki-e** [comment]
> What should an invocation look like?

I have no strong opinion about this, but I thought something like --no-clean=\<option>.

**Comment by smoelius** [comment]
> I have no strong opinion about this, but I thought something like --no-clean=<option>.

An argument I would make against that is the following. To me, `--no-clean=<option>` suggests `<option>` specifies what should _not_ be cleaned. If a user wants to keep everything except profraw files, `<option>` could be a long list. Also, it could hinder forward compatibility if new file types are stored in the `llvm-cov-target` directory.

An alternative I would propose is to add an option to `cargo llvm-cov clean`, perhaps `--profraw-only`.

Needless to say, I'll go with whatever you prefer, @taiki-e. :)
